python/tvm/relax/backend/contrib/cutlass.py

```diff
@@ -160,7 +162,7 @@ def _check_matmul(context: PatternCheckContext) -> bool:
 
     lhs_shape = lhs.struct_info.shape.values
     rhs_shape = rhs.struct_info.shape.values
-    return tvm.contrib.cutlass.build.is_shape_valid_for_cutlass_matmul(lhs_shape, rhs_shape)
+    return is_shape_valid_for_cutlass_matmul(lhs_shape, rhs_shape)
```

Comment:
Sorry I didn't notice that #15001 didn't revert the change in `contrib/cutlass.py` after the dep of `cutlass/build.py` to `contrib/cutlass.py` has been removed.

Why do we need local import here? I thought it is fine for `contrib/cutlass.py` to depend on `cutlass/build.py`.

Reply:
yeah it's fine, I have updated the PR